modules/claims_api/config/schemas/v2/request_bodies/disability_compensation/request.json

```diff
@@ -3,1080 +3,1093 @@
+                    "pattern": "^([-a-zA-Z0-9/']+( ?))*$",
+                    "example": "Jane Doe"
+                    },
+                "pointOfContactNumber": {
+                    "telephone": {
+                        "description": "Primary phone of Point of Contact.",
+                        "type": "string",
+                        "example": "1231234",
+                        "maxLength": 10
+                        },
+                    "internationalTelephone": {
+                        "description": "international phone of Point of Contact.",
+                        "type": "string",
+                        "example": "1231234",
+                        "maxLength": 10
+                    }
                 },
-                "city": {
-                  "description": "City Veteran resides in.",
-                  "type": "string",
-                  "pattern": "^([-a-zA-Z0-9'.#]([-a-zA-Z0-9'.# ])?)+$",
-                  "example": "Portland"
+                "currentlyHomeless": {
+                    "type": "object",
+                    "additionalProperties": false,
+                    "required": ["homelessSituationOptions", "otherDescription"],
+                    "properties": {
+                        "homelessSituationOptions": {
+                        "description": "Current state of the veteran's homelessness.",
+                        "type": "string",
+                        "default": "other",
+                        "enum": [
+                            "LIVING_IN_A_HOMELESS_SHELTER",
+                            "NOT_CURRENTLY_IN_A_SHELTERED_ENVIRONMENT",
+                            "STAYING_WITH_ANOTHER_PERSON",
+                            "FLEEING_CURRENT_RESIDENCE",
+                            "OTHER"
+                        ],
+                        "example": "FLEEING_CURRENT_RESIDENCE"
+                        },
+                        "otherDescription": {
+                        "description": "For a 'homelessSituationOptions' of value 'other', please provide additional comments on the living situation.",
+                        "type": "string",
+                        "maxLength": 500,
+                        "example": "other living situation"
+                        }
+                    }
                 },
-                "country": {
-                  "description": "Country Veteran resides in.  Value must match the values returned by the \"/countries\" endpoint on the [Benefits Reference Data API](https://developer.va.gov/explore/benefits/docs/benefits_reference_data?version=current).",
-                  "type": "string",
-                  "example": "USA"
+                "riskOfBecomingHomeless": {
+                    "type": "object",
+                    "additionalProperties": false,
+                    "required": [
+                        "livingSituationOptions",
+                        "otherDescription"
```

Comment:
From the ticket: "Update 'livingSituationOptions' and 'otherDescription' to be not required." So `otherDescription` needs to be removed from the required array.

Reply:
Both of these should be updated and fixed now.